TAScheduler/AcceptanceTests/test_login.py

```diff
@@ -0,0 +1,26 @@
+from django.test import TestCase, Client
+from TAScheduler.models import User
+
+class SuccessfulLogin(TestCase):
+    def setUp(self):
+        self.client = Client()
+        User.objects.create_user(email_address="user@example.com", password="password")
+
+    def test_login_with_valid_credentials(self):
+        response = self.client.post('/login/', {'email': 'user@example.com', 'password': 'password'})
```

Comment:
Our login page link is just "/" not "/login/" please make this fix throughout the file